activerecord/lib/active_record/attribute_methods.rb

```diff
@@ -181,7 +181,7 @@ def attribute_names
       #   "#{column_name} #{direction}"
       #   "#{column_name} #{direction} NULLS FIRST"
       #   "#{column_name} NULLS LAST"
-      COLUMN_NAME_ORDER_WHITELIST = /
+      COLUMN_NAME_ORDER_PERMIT_LIST = /
```

Comment:
This isn't even a list. I think `COLUMN_NAME_WITH_ORDER` would be sufficient here; likewise for `COLUMN_NAME` above.